services/intervention_service.py

```
@@ -79,7 +79,7 @@ def get_interventions(
             func.array(reason).label("reason"),
             case(
                 (
-                    PrescriptionB.concilia != None,
+                    and_(PrescriptionB.concilia != None, PrescriptionDrug.idDrug == 0),
```

Ruff (0.15.9)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)